modules/microshift-updating-rpms-ostree.adoc

```diff
@@ -4,9 +4,14 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="microshift-updates-rpms-ostree_{context}"]
-= Applying updates on an OSTree system
+= Applying updates on an rpm-ostree system
 
-To update {microshift-short} on an `rpm-ostree` system such as {op-system-ostree-first}, embed the new version of {microshift-short} on a new operating system image. The steps you use depends on how your existing deployment is set up. The following procedure outlines the general steps you can take, with links to the {op-system-ostree} documentation. The {op-system-ostree} documentation is your resource for specific details on building an updated operating system image. Back up and system rollback are automatic with this update type.
+To update {microshift-short} on an `rpm-ostree` system such as {op-system-ostree-first}, embed the new version of {microshift-short} on a new operating system image. Back up and system rollback are automatic with this update type.
+
+[IMPORTANT]
+====
+The steps you use depends on how your existing deployment is set up. The following procedure outlines the general steps you can take, with links to the {op-system-ostree} documentation. The {op-system-ostree} documentation is your resource for specific details on building an updated operating system image.
+====
 
 [NOTE]
```

Comment:
Try and avoid stacked admonitions as per the RH SSG: 

![Screenshot from 2024-08-28 16-53-26](https://github.com/user-attachments/assets/0001f30c-fa13-4472-af46-5be6d540b525)